sale_tier_validation/readme/CONFIGURE.md

```diff
@@ -3,3 +3,5 @@ To configure this module, you need to:
 1.  Go to *Settings \> Technical \> Tier Validations \> Tier
     Definition*.
 2.  Create as many tiers as you want for Sale Order model.
+
+If necessary, update the "Sale Report Print Block" under "Sales > Settings" to enable or disable block the printing of sale order report if the order is not validated  (enabled by default).
```

Comment:
Let's make the style of menu path presentation consistent with the existing content.

```suggestion
If necessary, update the "Block Printing of Unvalidated Quotations" under *Sales \>
Configuration \> Settings* to block the printing of quotation when the order has yet
to be validated (disabled by default).
```